packages/utils/src/groupInMap.ts

```diff
@@ -18,35 +10,23 @@ const DELETE = Symbol("DELETE")
  * @param keyGetter A function that extracts the key for each item.
  * @param projection Projection function for each group.
  */
-export const groupInMap = <T, K, V>(
-  keyGetter: (x: T) => K,
-  projection: (x: GroupedObservable<K, T>) => Observable<V>,
-) => (source$: Observable<T>): Observable<Map<K, V>> => {
+export const groupInMap = <K, V>() => (
+  source$: Observable<GroupedObservable<K, V>>,
+): Observable<Map<K, V>> => {
   const res = new Map<K, V>()
```

Comment:
I think different subscriptions will use the same reference of this map... Maybe the scan overload you made with a lazy initial value can come in handy?

Reply:
Very good point, thanks for pointing that out!